<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionDialog</class>
 <widget class="QDialog" name="QuestionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Questions</string>
  </property>
  <widget class="QLabel" name="testQuestionLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="possibilty1Button">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>90</y>
     <width>451</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="possibilty2Button">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>140</y>
     <width>451</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="possibilty3Button">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>190</y>
     <width>451</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="possibilty4Button">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>240</y>
     <width>451</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>310</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
